src/main/java/com/checkmarx/jenkins/CxScanBuilder.java

```diff
@@ -2433,8 +2436,8 @@ public FormValidation doTestConnection(@QueryParameter final String serverUrl, @
          * browser.
          */
         @POST
-        public FormValidation doCheckScaSASTProjectID(@QueryParameter String value, @QueryParameter String scaSASTProjectFullPath) {
-            Jenkins.getInstance().checkPermission(Item.CONFIGURE);
+        public FormValidation doCheckScaSASTProjectID(@QueryParameter String value, @QueryParameter String scaSASTProjectFullPath,@AncestorInPath Item item) {
+                item.checkPermission(Item.CONFIGURE);
```

Comment:
```suggestion
                if (item == null) {
                        return FormValidation.ok();
                }
                item.checkPermission(Item.CONFIGURE);
```
Please check if the context is not null